app/assets/javascripts/discourse/app/controllers/user-card.js

```diff
@@ -1,10 +1,9 @@
-import { inject as service } from "@ember/service";
 import Controller, { inject as controller } from "@ember/controller";
 import DiscourseURL, { userPath, groupPath } from "discourse/lib/url";
 
 export default Controller.extend({
+  application: controller(),
```

Comment:
Is the `application` controller injection used in `user-card`?

Reply:
No, good catch!